Backoffice/EventSubscriber/BlockTransverseCacheSubscriber.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\EventSubscriber;
+
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use OpenOrchestra\ModelInterface\NodeEvents;
+use OpenOrchestra\ModelInterface\Event\NodeEvent;
+use OpenOrchestra\DisplayBundle\Manager\CacheableManager;
+use OpenOrchestra\BaseBundle\Manager\TagManager;
+
+/**
+ * Class BlockTransverseCacheSubscriber
+ */
+class BlockTransverseCacheSubscriber implements EventSubscriberInterface
```

Comment:
@alavieille : ``TransverseBlockCacheSubscriber`` instead of ``BlockTransverseCacheSubscriber``